website/docs/utilities/popover-primitive/partials/code/component-api.md

```diff
@@ -0,0 +1,123 @@
+## Component API
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="isOpen" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Controls if the popover should be rendered initially opened.
+    <br />
+    <br />
+    _Notice: in this case the popover can't be dismissed via `esc` or `click outside` until the end user has interacted with it (it's in a "manual" state)._
+  </C.Property>
+  <C.Property @name="enableSoftEvents" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Assigns "soft" events listeners (`mouseEnter/Leave` + `focusIn/Out`) to the toggle, to control the visibility of the popover content.
+  </C.Property>
+  <C.Property @name="enableClickEvents" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Assigns a "click" event listener (`onClick`) to the toggle, to control the visibility of the popover content.
```

Comment:
I don't think we need the quotes here. 

```suggestion
    Assigns a click event listener (`onClick`) to the toggle, to control the visibility of the popover content.
```

Reply:
Agree. I think there's an overuse of quotes overall in some of the text here.